### PR TITLE
Clean up duplicate definitions in POWER8 kernels and fix power10 option passing

### PR DESCRIPTION
**Pull request by martin-frbg**
for #2911

### CODE REVIEW

File: Makefile.power [hunk 1]
```diff
@@ -10,7 +10,7 @@ USE_OPENMP = 1
 endif
 
 ifeq ($(CORE), POWER10)
-COMMON_OPT += -Ofast -mcpu=power10 -mtune=power10 -mvsx  -fno-fast-math
+COMMON_OPT += -Ofast -mcpu=power10 -mtune=power10 -mvsx -mmma  -fno-fast-math
```
Comment (RajalakshmiSR) [inline Makefile.power]: We dont need to add -mmma as -mcpu=power10 takes care of enabling mma options by default.
If we add the binutils check, we don't need this change here.

Comment (martin-frbg) [reply Makefile.power]: Yes, this was only my initial reaction to the issue ticket - the actual problem must be that this line has no effect when COMMON_OPT is passed in via the `make` command line.

File: kernel/power/strmm_kernel_16x8_power8.S [hunk 2]
```diff
@@ -12,7 +12,7 @@ the documentation and/or other materials provided with the
 distribution.
 3. Neither the name of the OpenBLAS project nor the names of
 its contributors may be used to endorse or promote products
-derived from this software without specific prior written permission.
+derived from this software without specific prior written permission.
```
Comment (h-vetinari) [inline kernel/power/strmm_kernel_16x8_power8.S]: Spurious spaces here...

Comment (martin-frbg) [reply kernel/power/strmm_kernel_16x8_power8.S]: Thanks, probably introduced them by using the github web editor. Wish we were back at that level of "problems" though.

**Pull request merged**